changelog.txt

```diff
@@ -525,7 +525,7 @@
 = 1.12.0 - 2022-03-29 =
 * Add - Additional data points for tracker snapshot.
 * Add - Enables merchants to select multiple countries as their audience when creating a Google Ads campaign (Smart Shopping Campaign).
-* Add - Google Listings And Ads product attributes icon.
+* Add - Google for WooCommerce And Ads product attributes icon.
```

Comment:
The `And Ads` wasn't replaced here.

I was actually wondering should we do the rename in changelog? As the changelog records what we've done in the past and the name Google Listings & Ads would be the history as well.

Reply:
Right @ianlin I restore the old changes in the changelog and just replaced the title. Can you take another look?